src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,114 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final int minAge = 18;
+    private final int minPasswordLength = 6;
+    private final String login = "Ivanov";
+    private final String password = "Hello_World";
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
```

Comment:
```suggestion
    private RegistrationService registrationService;
```
move initialization to a method with annotation BeforeAll or BeforeClass ( depends on junit version)